Tests/FilesTest.cs

```diff
@@ -0,0 +1,257 @@
+﻿using System;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Microsoft.SharePoint.Client;
+using System.Management.Automation.Runspaces;
+using System.Collections;
+using System.Linq;
+using System.Management.Automation;
+using System.Text;
+
+namespace SharePointPnP.PowerShell.Tests
+{
+    [TestClass]
+    public class FilesTests
+    {
+        private const string targetLibraryRelativeUrl = "/sites/dev/Shared%20Documents";
```

Comment:
Hi Kevin! You have hardcoded URLs in this test. Please refactor this to use dynamic sites (for instance, automatically create two subwebs in the site collection specified in the app.config and use those for testing and remove them afterwards. Use unique names/guids for those subwebs as other tests might create subwebs too)